newsroom/settings.py

```diff
@@ -48,12 +57,12 @@ def update_values():
     return flask.jsonify(updates)
 
 
-def get_initial_data(setting_key=None):
-    data = get_setting(setting_key=setting_key, include_audit=True)
+async def get_initial_data(setting_key=None):
+    data = await get_setting(setting_key=setting_key, include_audit=True)
     return data
 
 
-def get_setting(setting_key=None, include_audit=False):
+async def get_setting(setting_key=None, include_audit=False):
```

Comment:
Looks like `await` is not used anywhere in this function, therefor there is no need to make this function async. You would then be able to remove all of the `await get_setting` changes you have added